score/scripts/download_cbg.py

```diff
@@ -5,70 +5,64 @@
 import json
 from pathlib import Path
 
+from utils import get_state_fips_codes
+
 data_path = Path.cwd() / "data"
 
 with requests.Session() as s:
     # the fips_states_2010.csv is generated from data here
     # https://www.census.gov/geographies/reference-files/time-series/geo/tallies.html
-    fips_csv_path = data_path / "fips_states_2010.csv"
-    with open(fips_csv_path) as csv_file:
-        csv_reader = csv.reader(csv_file, delimiter=",")
-        line_count = 0
-        for row in csv_reader:
-            if line_count == 0:
-                line_count += 1
-            else:
-                fips = row[0].strip()
-
-                # check if file exists
-                shp_file_path = data_path.joinpath(
-                    "census", "shp", fips, f"tl_2010_{fips}_bg10.shp"
-                )
-                if not os.path.isfile(shp_file_path):
-                    print(f"downloading {row[1]}")
+    state_fips_codes = get_state_fips_codes()
+    for fips in state_fips_codes:
+        # check if file exists
+        shp_file_path = data_path.joinpath(
+            "census", "shp", fips, f"tl_2010_{fips}_bg10.shp"
```

Comment:
later on the other script uses the "score" directory - after this change I'm seeing "No GeoJson files found. Please run scripts/download_cbg.py first"

Reply:
resolved - i had run an earlier version of the script on an earlier version of the branch